qml/MonthComponent.qml

```diff
@@ -45,6 +45,7 @@ Item{
 
     signal monthSelected(var date);
     signal dateSelected(var date);
+    signal pressAndHoldAt(var date, bool allDay);
```

Comment:
This seems to be unused, why are you adding it?

Reply:
Actually I wanted to do it as its done for day and week view. But there the signal is caught and reemitted (maybe thats needed because there a loader is used...?) which seems to be not necessary here, so I changed it accordingly and forgot to remove the signal.